sdk/schemaregistry/azure-data-schemaregistry-apacheavro/CHANGELOG.md

```diff
@@ -11,6 +11,16 @@
 ### Other Changes
 
 
+## 1.1.17 (2024-05-28)
+
+### Other Changes
+
+#### Dependency Updates
+
+- Upgraded `azure-core` from `1.48.0` to version `1.49.0`.
+- Upgraded `azure-data-schemaregistry` from `1.4.5` to version `1.5.0-beta.1`.
```

Comment:
This should mention a stable version like `1.4.6` or whatever was recently released for Schema Registry.